resources/wikia/modules/krux.js

```diff
@@ -0,0 +1,64 @@
+/*global Krux,define*/
+window.Krux || ((Krux = function () {
+	Krux.q.push(arguments);
+}).q = []);
+
+define('wikia.krux', ['wikia.window'], function (win) {
+	'use strict';
+
+	var maxNumberOfKruxSegments = 27;
+
+	function load(confid) {
```

Comment:
In order to call load() on each PV we do have to do it explicitly (like Mercury does with ComScore and QuantCast). However, Krux stays loaded among page reloads. So idea for splitting load() into two methods is OK!